spec/lib/tasks/backfill_sponsor_id_rake_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+require 'rake'
+
+RSpec.describe 'in_person_enrollments:backfill_sponsor_id rake task' do
+  let!(:task) do
+    Rake.application.rake_require 'tasks/backfill_sponsor_id'
+    Rake::Task.define_task(:environment)
+    Rake::Task['in_person_enrollments:backfill_sponsor_id']
+  end
+
+  subject(:invoke_task) do
+    actual_stderr = $stderr
+    proxy_stderr = StringIO.new
+    begin
+      $stderr = proxy_stderr
+      task.reenable
+      task.invoke
+      proxy_stderr.string
+    ensure
+      $stderr = actual_stderr
+    end
+  end
+
+  let(:pending_enrollment) { create(:in_person_enrollment, :pending, :with_nil_sponsor_id) }
+  let(:expired_enrollment) { create(:in_person_enrollment, :expired, :with_nil_sponsor_id) }
+  let(:failed_enrollment) { create(:in_person_enrollment, :failed, :with_nil_sponsor_id) }
+  let(:enrollment_with_service_provider) do
+    create(:in_person_enrollment, :with_service_provider, :with_nil_sponsor_id)
+  end
+  let(:enrollment_with_sponsor_id) { create(:in_person_enrollment) }
+
+  before do
+    allow(IdentityConfig.store).to receive(:usps_ipp_sponsor_id).and_return('31459')
+    # binding.pry
```

Comment:
let's remove this?
```suggestion
```

Reply:
Thanks for catching that!  I removed it in commit [715d76a](https://github.com/18F/identity-idp/pull/10988/commits/715d76a48a112641b61ddeca981b8b47aa2029a8).